resources/manifest.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: manifests.package-operator.run/v1alpha1
+kind: PackageManifest
+metadata:
+  name: managed-openshift-release-bundle
+spec:
+  scopes:
+  - Namespaced
+  - Cluster
+  phases:
+  - name: release-bundle-configmap
```

Comment:
I know this is a placeholder, but phases are meant to group resources, so I'd expect a phase name to be something like "first", "second", ... or "apis", "workloads", ...

If you don't need coordinated rollout, you can just have a simple phase and annotate all objects with the same phase name.

Reply:
I think what I was initially going for was each component has it's own phase (which would allow us to set priority based on what should be installed first). I can't group everything together into a single phase though (namespaces need to be created before other resources, so they are separated into their own phase). At the moment, we don't have anything real in the bundle so yeah this is just a placeholder.

Reply:
Unless of course you want a single object in a phase, then you're have the phase "release-bundle-configmap" and another phase would be "all-the-rest", but that doesn't seem to be what you need here :)

Reply:
yes yes, agreed... we will likelly have a first phase "namespaces" and then one or a few more... but not necessarily one phase per resource.